.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,21 @@
+rebaseMergeAllowed: false
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: master
+  isAdminEnforced: true
+  requiredStatusCheckContexts:
+    - 'ci/circleci: build'
+    - 'ci/circleci: protobufjs-load-test'
+    - 'ci/circleci: push-generated-source'
+    - 'ci/circleci: test'
+    - 'ci/circleci: typescript-smoke-test'
+    - 'cla/google'
+    - 'codecov/patch'
+    - 'codecov/project'
```

Comment:
Hi, can we please not include these are required checks? There is some weird flakey bug that drops our coverage on an uncoverable line which makes codecov unhappy. Maybe there is a codecov config thing I could change, but idk.

Reply:
Ironically, I wouldn't have been able to merge this PR 😆 Fixed!

Reply:
Thank you!!